docs/source/getting_started.rst

```diff
@@ -5,11 +5,12 @@ Getting started
 
 .. code-block:: shell
 
-    $ virtualenv my_project
-    $ source my_project/bin/activate
-    $ pip install ramses
-    $ pcreate -s ramses_starter my_project
+    $ mkdir my_project 
     $ cd my_project
+    $ {virtualenv|pyvenv} my_venv
```

Comment:
Might be a bit confusing for newbies. I would keep it simple:`$ virtualenv venv`, like in the "The Hitchhiker's Guide".